services/company.service.ts

```diff
@@ -0,0 +1,38 @@
+import { Company } from "../models";
+import { ICompany } from "../models/interface/company.model.interface";
+
+export class CompanyService {
+
+    public async createCompany(company: ICompany) {
+        try {
+
+            const companyResponse = await new Company(company);

```

Comment:
Where you are generating tenantId using `uuid`?

Reply:
its generating in company model